static/app/views/dashboardsV2/data.tsx

```diff
@@ -11,12 +15,13 @@ export const EMPTY_DASHBOARD: DashboardDetails = {
   widgets: [],
 };
 
-export const DASHBOARDS_TEMPLATES: DashboardDetails[] = [
+export const DASHBOARDS_TEMPLATES: DashboardTemplate[] = [
   {
     id: 'default-template',
     dateCreated: '',
     createdBy: undefined,
-    title: t('Default'),
+    title: t('General'),
+    description: t('Various Frontend & Backend Widgets'),
```

Comment:
nit: can use `and` for consistency
```suggestion
    description: t('Various Frontend and Backend Widgets'),
```